index.html

```diff
@@ -2,84 +2,130 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="stylesheet" href="https://use.typekit.net/yri5dks.css" />
     <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="scss/vendor/reset.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
 
     
   <link rel="stylesheet" href="/assets/style.css" /></head>
   <body>
-    <div class="start-screen-block">
-      <header>
-        <ul>
-          <li><a href="#">HOME</a></li>
-          <li><a href="#">PORTFOLIO</a></li>
-          <li><a href="#">BLOG</a></li>
-          <li><a href="#">PAGES</a></li>
-          <li><a href="#">ABOUT US</a></li>
-          <li><a href="#">CONTACT</a></li>
-        </ul>
-      </header>
-      <div class="header-logo">
-        <img src="src/img/logo.png" style="width:50px;height:50px" />
-        <p>Expire</p>
-      </div>
+    <div class="container">
+      <div class="start-screen-block">
+        <header>
+          <ul>
+            <li><a href="#">home</a></li>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">blog</a></li>
+            <li><a href="#">pages</a></li>
+            <li><a href="#">about us</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+        </header>
+        <div class="header-logo">
+          <img src="src/img/logo.png" />
+          <p>Expire</p>
+        </div>
 
-      <div class="header-slider">
-        <div class="arrows">
-          <img
-            src="src/img/left.png"
-            style="margin:120px -1px;position:absolute;"
-          />
-          <img
-            src="src/img/right.png"
-            style="margin-top:120px;position:absolute;right:1px;"
-          />
+        <div class="header-slider">
+          <div class="arrows">
+            <img src="src/img/left.png" class="left-arrow" />
+            <img src="src/img/right.png" class="right-arrow" />
+          </div>
+          <p class="slider-headline">
+            Expire
+          </p>
+          <img src="src/img/line.png" class="line" />
+          <p class="slider-paragraph">
+            Professionaly designed, carefully made for your enjoyement
+          </p>
+          <div class="slider-buttons">
+            <button class="explore-btn">explore</button>
+            <button class="learn-btn">
```

Comment:
I think we should add the common button class, and then if we need modify we can add a new class
(for all similar buttons)

Reply:
Done